topk-js/src/data/utils.rs

```diff
@@ -0,0 +1,29 @@
+use napi::bindgen_prelude::*;
+
+pub unsafe fn is_napi_integer(env: napi::sys::napi_env, napi_val: napi::sys::napi_value) -> bool {
```

Comment:
I don't think this function is sound. if I have `[1.0, 2.0, 3.0]` it will consider all of these as ints

Reply:
we checked and it is sound